### PR TITLE
#9 feature(Buyer should be able to provide a review)

### PR DESCRIPTION
**Pull request by nili-alu**
**What does this PR do?**

- Buyer should be able to provide a review
- Seller should be able to see reviews on each product
- Seller should be able to delete irrelevant feedback 

**Description of Task to be completed**

Buyer should provide review only when he/she has bought something 
Only Seller can delete irrelevant feedback(s)

**How should this be manually tested?**

**1. In terminal**
checkout to this branch

- `git checkout ft-buyer-provide-review-9`

- ` npm i `

- ` rpm run dev `

**2. In postman**

--> add review :  past this url : http://localhost:3000/api/review/:id and use POST method. **Note:** the id is the id of product you want to review, and make sure you passed buyer token

in body pass: 

-  "rating": your rating integer here between 1 and 5

-   "feedback": "your feedback here"

--> Get review:  http://localhost:3000/api/review   **Note**: make sure you passed buyer token
--> Delete review:  http://localhost:3000/api/review/:id  **Note**: make sure you passed seller token
--> Get product by id and see its reviews: http://localhost:3000/api/product/:id

What are the relevant pivotal tracker stories?
- [Card 9:](https://trello.com/c/4biwZh3T/9-buyer-should-be-able-to-provide-a-review)

### CODE REVIEW

**Review by ALVINdimpos — APPROVED** [review]

**Review by NTElissa — APPROVED** [review]
good progress

**Review by Muhet — APPROVED** [review]
keep it up

File: src/controller/reviewController.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        // Handle database errors
+        console.error(error);
+        return res.status(500).json(jsend.error('Failed to save review'));
+    }// end of addReview
+}
+
+// //get all reviews
+export const getReviews = async (req, res) => {
+    try {
+        // const product_id= req.params.id;
```
Comment (chanceDira) [inline src/controller/reviewController.js]: remove then code if we are not using them.

**Review by chanceDira — CHANGES_REQUESTED** [review]
Nice work @nili-alu just one thing

File: src/controller/reviewController.js (hunk 1 above)
Comment (nili-alu) [reply src/controller/reviewController.js]: done

**Review by chanceDira — APPROVED** [review]

**Pull request merged**